src/main/scala/org/broadinstitute/dsde/agora/server/webservice/ServiceHandlerProps.scala

```diff
@@ -0,0 +1,10 @@
+
+package org.broadinstitute.dsde.agora.server.webservice
+
+import akka.actor.Props
+
+trait ServiceHandlerProps {
```

Comment:
Since we're not overriding with MethodsServiceHandlerProps and ConfigsServiceHandlerProps, and the underlying handlers have been made generic, does this need to be it's own class?  Or can it become part of AgoraService?